monkey/monkey_island/cc/ui/src/components/report-components/ransomware/BreachSection.tsx

```diff
@@ -51,7 +79,7 @@ function getBreachSectionBody(machines) {
 
 function getMachine(machine) {
   return (
-    <li key={machine['hostname']}>
+    <li key={machine['hostname']+''+machine['start_time']}>
```

Comment:
Why add an empty string (`''`) between the two?

Reply:
I wanted the keys to be specific for each item in the list and just make sense.

Reply:
Why not just do 
```suggestion
    <li key={machine['hostname']+machine['start_time']}>
```